src/components/micro-frame-sse/README.md

```diff
@@ -0,0 +1,254 @@
+<h1 align="center">
+  <!-- Logo -->
+  <br/>
+  micro-frame-sse & micro-frame-slot
+	<br/>
+</h1>
+
+<p align="center">
+  SSE stream support for micro-frame
+</p>
+
+# Installation
+
+```console
+npm install @micro-frame/marko
+```
+
+# How it works
+
+Similar to `<micro-frame>`, the package exposes `<micro-frame-sse>` and `<micro-frame-slot>` tags in order to initiate SSE stream and placement of streaming content.
+
+It works the same way as [micro-frame](../../../README.md) when used in browser vs in node server.
+
+## Why
+
+In addition to the benefits from [micro-frame](../../../README.md), `<micro-frame-sse>` allows short living server-sent event to steam content into designated slots within single http request. In another word, `micro-frame-sse` enables multiplexing of `micro-frame`.
+
+Considering SSE is only one kind of streaming formats, `micro-frame` could potentially support all kinds of streaming formats like `json-stream` with a proper `parser`.
+
+![stream explained](stream_explained.jpg)
+
+# Example
+
+`micro-frame-sse` tag is required to put into earlier stage of page rendering (mainly at beginning of the page), who will initiate the SSE request as soon as page starts. `micro-frame-slot` is where content being injected and can be placed at any desired location.
+
+```marko
+<micro-frame-sse
+  src="sse-source"
+  name="unique_name"
+  read(ev) {
+    // assuming data is in JSON format
+    const data = JSON.parse(ev.data);
+    const slotId = data.slotId, html = data.content;
+    return [slotId, html];
+  }
+>
+  <@catch|err|>
+      Uh-oh! ${err.message || err}
```

Comment:
I almost think we should remove `<@catch>` and `<@loading>` from the `micro-frame-sse` tag and push that logic down to the individual slots.

Since this tag merely kicks of the connection I'm not sure it's ever going to be in a useful place in the document to display something to the user.

Reply:
micro-frame-sse error will also be emitted to individual slots and micro-frame-slot has its own `catch` and `loading`. I feel it doesn't hurt to have these options here and users can ignore these if they don't need.

Reply:
I have concerns around runtime size overhead with these additional apis and the maintenance burden.

Typically like to avoid adding API's without a real world use case in mind and/or customer request.